lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc

```diff
@@ -278,7 +279,7 @@ function dosomething_reportback_views_default_views() {
   $handler->display->display_options['menu']['context_only_inline'] = 0;
 
   /* Display: Page */
-  $handler = $view->new_display('page', 'Page', 'page_1');
+  $handler = $view->new_display('page', 'Page', 'reportback_review');
```

Comment:
![screen shot 2016-02-23 at 9 33 01 am](https://cloud.githubusercontent.com/assets/897368/13254702/7eb2f3ea-da10-11e5-95b8-54b9be2dc5c2.png)

and its only applied to that specific page